beacon_node/beacon_chain/src/kzg_utils.rs

```diff
@@ -13,7 +13,8 @@ pub fn validate_blob<T: EthSpec>(
     blob: &Blob<T>,
     kzg_commitment: KzgCommitment,
     kzg_proof: KzgProof,
-) -> Result<bool, KzgError> {
+) -> Result<(), KzgError> {
+    let _timer = crate::metrics::start_timer(&crate::metrics::KZG_VERIFICATION_SINGLE_TIMES);
```

Comment:
Looks like we're also doing the timer in `blob_verification`, maybe remove the other one? 
https://github.com/sigp/lighthouse/blob/e5a0e2c500f756ef7d2abf556e3371cc5dd96d3e/beacon_node/beacon_chain/src/blob_verification.rs#L305